sips/sip-030/sip-030-wallet-interface.md

```diff
@@ -0,0 +1,527 @@
+# Preamble
+
+SIP Number: `030`
+
+Title: Integration of a Modern Stacks Wallet Interface Standard
+
+Authors: [aryzing](https://github.com/aryzing), [janniks](https://github.com/janniks), [kyranjamie](https://github.com/kyranjamie),  [m-aboelenein](https://github.com/m-aboelenein)
+
+Type: Standard
+
+Status: Draft
+
+Created: 10 October 2023
+
+License: BSD 2-Clause
+
+# OPEN QUESTIONS
+
+<!-- todo: remove section before merge -->
+
+- [ ] Should a global single `window.` object be used, or should provider discovery be handled complely differently?
+- [ ] Rename `stx_transferFt` to include SIP010 somewhere?
+- [ ] Should listeners have params? (e.g. txid for tx mined event)
+
+# Abstract
+
+This SIP proposes common RPC methods to use for the Stacks blockchain's "Connect" and "Auth" systems.
+The goal is to replace the current Connect interface, primarily used by web applications to connect with browser extensions and mobile apps with a more straightforward protocol.
+This proposal's goal is to standardize JSON compatible interfaces for use with wallet interfaces.
+
+# Motivation
```

Comment:
'Motivation' section should be renamed 'Introduction'?: The SIP Specification requires an 'Introduction' section that provides a high-level summary of the problem(s) and how the proposal solves them.